### PR TITLE
build: add license checker workflow

### PR DESCRIPTION
**Pull request by Wwwsylvia**
Resolves: #9

### CODE REVIEW

**Comment by Wwwsylvia** [comment]
An example of how it runs: https://github.com/Wwwsylvia/oras-credentials-go/actions/runs/4476341442

File: .github/licenserc.yml
Comment (shizhMSFT) [inline .github/licenserc.yml:1]: It is better to newline at end of file

Comment (Wwwsylvia) [reply .github/licenserc.yml:1]: Updated.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**